nix/hsie/default.nix

```diff
@@ -28,8 +28,8 @@ let
         mkdir -p $out/bin
         ln -s $hsie $out/bin/$name
       '';
-  bashCompletion =
+  bash-completion =
```

Comment:
The updated nixpkgs came with deprecation warnings against spelling it `bashCompletion` :shrug:

Reply:
That's weird. it's not about using a camelcase key right? Just about this particular identifier.

Reply:
Yes, that particular one. It was changed from a warning to an exception here: https://github.com/NixOS/nixpkgs/pull/161146